README.md

```diff
@@ -186,30 +186,44 @@ pre-commit: `brew install circleci`
 
 To set up docker on your local machine:
 
-```sh
+```console
 brew cask install docker
 brew install docker-completion
 ```
 
 Now you will need to start the Docker service: run Spotlight and type in
 'docker'.
 
-### Setup: PostgreSQL
+#### docker-compose
 
-To run postgres in Docker, first you will need the image:
+To run the app in Docker locally, we are using [`docker-compose`](https://docs.docker.com/compose/):
 
-```sh
-docker pull postgres:11.6
+```console
+brew install docker-compose
+brew install docker-compose-completion  # optional
+```
+
+Now, you can stand up the app & database locally in containers.  The app server
+will stand up and migrations from your local filesystem will be run against the
+db automatically:
+
+```console
+$ docker-compose up --detach
+Starting app_db_1 ... done
+Starting app_db_clean_1 ... done
+Starting app_db_migrate_1 ... done
+Starting app_easi_1       ... done
+$ curl http://localhost:8080/api/v1/healthcheck
+{"status":"pass","datetime":"APPLICATION_DATETIME","version":"APPLICATION_VERSION","timestamp":"APPLICATION_TS"}
 ```
 
-Now, you can run postgres locally in a container:
+To take it down:
```

Comment:
Would it help to add a couple quick notes about `docker-compose build` and `docker-compose down` and use cases for using those? `docker-compose kill` will stop the currently running containers; a subsequent `docker-compose up` would restart those same stopped containers. App engineers will likely have a use case where they want to rebuild the image and deploy a new container if they've made code changes.